projects/kit/components/slider/slider.component.ts

```diff
@@ -100,6 +102,11 @@ export class TuiSliderComponent {
         return isEdgeOlderThan(CHROMIUM_EDGE_START_VERSION, this.userAgent);
     }
 
+    @tuiPure
+    get hasKeySteps(): boolean {
+        return Boolean(this.injector.get(TuiSliderKeyStepsDirective, null));
```

Comment:
I cannot just declare this constant as 
```ts
export class TuiSliderComponent {
    private readonly hasKeySteps = Boolean(
        this.injector.get(TuiSliderKeyStepsDirective, null)
    );
    // ...
}
```

because it will cause
```
ERROR Error: Circular dep for TuiSliderComponent
```